addon/components/document-grid.hbs

```diff
@@ -56,6 +67,17 @@
           {{/each}}
         {{/if}}
       </div>
+      {{#if this.isDragOver}}
+        <button
+          class="[object Object],uk-width-1
```

Comment:
Please remove the `[object Object]`s and prettify

Reply:
Nice catch! I wonder where this came from. Especially twice.

I removed it and put everything through prettier.